proto/cosmwasm/wasm/v1/authz.proto

```diff
@@ -57,7 +57,7 @@ message MaxCallsLimit {
 // MaxFundsLimit defines the maximal amounts that can be sent to the contract.
 // Since: wasmd 0.30
 message MaxFundsLimit {
-  option (cosmos_proto.implements_interface) = "ContractAuthzLimitX";
+  option (cosmos_proto.implements_interface) = "cosmwasm.wasm.ContractAuthzLimitX";
```

Comment:
1. [registered interface here](https://github.com/cosmology-tech/wasmd/blob/5c39daee99b838482ec1bdd980bfa0a56213ec34/x/wasm/types/codec.go#L87)
2. [interface defined here](https://github.com/cosmology-tech/wasmd/blob/5c39daee99b838482ec1bdd980bfa0a56213ec34/x/wasm/types/authz.go#L198)

would this make it `cosmwasm.wasm.ContractAuthzLimitX`?